nengo/utils/nco.py

```diff
@@ -71,6 +71,11 @@ def read(self, size=None):
         self.max_size -= size
         return self.fileobj.read(size)
 
+    def readinto(self, b):
```

Comment:
What is this function used for?

Reply:
Reads the contents of a file into a pre-allocated byte array ([docs](https://docs.python.org/3/library/io.html#io.RawIOBase.readinto)). I don't think we use it, but TravisCI was raising an error for having the audacity to open a file object that didn't implement `readinto`.